draft-ietf-ppm-dap.md

```diff
@@ -1380,7 +1380,10 @@ This message consists of:
 Finally, the Leader sends a PUT request to
 `{helper}/tasks/{task-id}/aggregation_jobs/{aggregation-job-id}`. The payload
 is set to `AggregationJobInitReq` and the media type is set to
-"application/dap-aggregation-job-init-req".
+"application/dap-aggregation-job-init-req". After receiving HTTP status 202
+Created, the Leader proceeds to poll GET requests to the aggregation job URI
```

Comment:
```suggestion
"application/dap-aggregation-job-init-req". After receiving HTTP status 201
Created, the Leader proceeds to poll GET requests to the aggregation job URI
```

Reply:
Further, the use of the verb "poll" is awkward: what's being polled is the aggregation job, not the GET request. The GET request is the representation of polling an aggregation job. Something like "the Leader polls the aggregation job by sending GET requests to the aggregation job URI" is better.